### ISSUE TITLE
www.digifail.com is down

### ISSUE THREAD

**Issue by stappersg** [MS3FGX/Bluelog#13]
Hi,

For what it is worth:

Website www.digifail.com  is down.  That website is in the "github project about box".

I do hope that the website can be fixed (or that the "upper right corner of GH project" gets a valid alternative).

**Comment by MS3FGX** [comment]
Yeah, my personal web server has been offline for a bit as I transition to a new hardware setup. Unfortunately like most of my projects, it's taken a bit longer than I anticipated.

That said, you aren't really missing anything. The README here on GitHub is pretty much a 1:1 copy of what was on the page, with a few relatively tribal exceptions. For instance, the page has screenshots of all the different live interfaces.

The site/server will certainly come back up at some point, but I don't have an ETA right now.

**Comment by stappersg** [comment]
Ah, OK.

(This issue now serves as _No need for further reports on website down_)
